malariagen_data/anopheles.py

```diff
@@ -3159,7 +3158,7 @@ def plot_njt(
         sample_sets: Optional[base_params.sample_sets] = None,
         sample_query: Optional[base_params.sample_query] = None,
         sample_indices: Optional[base_params.sample_indices] = None,
-        site_mask: Optional[base_params.site_mask] = None,
+        site_mask: Optional[base_params.site_mask] = base_params.DEFAULT,
```

Comment:
Will cause behaviour change for `plot_njt()` when `site_mask` parameter is not given.